src/docs/templates/reference_commands.md

```diff
@@ -0,0 +1,4 @@
+# Commands
```

Comment:
For each reference page, there's a corresponding template in `src/docs/templates`, which is prepended to the generated content.